Examples/BookingSystem.AspNetCore/Stores/OrderStore.cs

```diff
@@ -24,7 +23,11 @@ public class AcmeOrderStore : OrderStore<OrderTransaction, OrderStateContext>
         public override bool CustomerCancelOrderItems(OrderIdComponents orderId, SellerIdComponents sellerId, OrderIdTemplate orderIdTemplate, List<OrderIdComponents> orderItemIds)
         {
             //throw new OpenBookingException(new CancellationNotPermittedError());
-            return FakeBookingSystem.Database.CancelOrderItems(orderId.ClientId, sellerId.SellerIdLong ?? null  /* Hack to allow this to work in Single Seller mode too */, orderId.uuid, orderItemIds.Select(x => x.OrderItemIdLong.Value).ToList(), true);
+            return FakeBookingSystem.Database.CancelOrderItems(
+                orderId.ClientId,
+                sellerId.SellerIdLong ?? null /* Hack to allow this to work in Single Seller mode too */,
```

Comment:
Note have restored the ` ?? null` (that was removed by linter) for the "hack" to make it clear what's happening. We should remove the hack entirely in future, but for now suggest we keep the clarity.